src/m365/pp/commands/card/card-list.spec.ts

```diff
@@ -0,0 +1,202 @@
+          "_modifiedonbehalfby_value": null,
+          "componentstate": 0,
+          "statecode": 0,
+          "name": "DummyCard",
+          "versionnumber": 3044006,
+          "utcconversiontimezonecode": null,
+          "cardid": "69703efe-4149-ed11-bba2-000d3adf7537",
+          "publishdate": null,
+          "_createdonbehalfby_value": null,
+          "_modifiedby_value": "7d48edd3-69fd-ec11-82e5-000d3ab87733",
+          "createdon": "2022-10-11T08:52:12Z",
+          "overwritetime": "1900-01-01T00:00:00Z",
+          "_owningbusinessunit_value": "2199f44c-195b-ec11-8f8f-000d3adca49c",
+          "hiddentags": null,
+          "description": " ",
+          "appdefinition": "{\"screens\":{\"main\":{\"template\":{\"type\":\"AdaptiveCard\",\"body\":[{\"type\":\"TextBlock\",\"size\":\"Medium\",\"weight\":\"bolder\",\"text\":\"Your card title goes here\"},{\"type\":\"TextBlock\",\"text\":\"Add and remove element to customize your new card.\",\"wrap\":true}],\"actions\":[],\"$schema\":\"http://adaptivecards.io/schemas/1.4.0/adaptive-card.json\",\"version\":\"1.4\"},\"verbs\":{\"submit\":\"echo\"}}},\"sampleData\":{\"main\":{}},\"connections\":{},\"variables\":{},\"flows\":{}}",
+          "statuscode": 1,
+          "remixsourceid": null,
+          "sizes": null,
+          "_owningteam_value": null,
+          "coowners": null,
+          "_createdby_value": "7d48edd3-69fd-ec11-82e5-000d3ab87733",
+          "_ownerid_value": "7d48edd3-69fd-ec11-82e5-000d3ab87733",
+          "publishsourceid": null,
+          "timezoneruleversionnumber": null,
+          "iscustomizable": {
+            "Value": true,
+            "CanBeChanged": true,
+            "ManagedPropertyLogicalName": "iscustomizableanddeletable"
+          },
+          "owninguser": {
+            "azureactivedirectoryobjectid": "88e85b64-e687-4e0b-bbf4-f42f5f8e574c",
+            "fullname": "Contoso Admin",
+            "systemuserid": "7d48edd3-69fd-ec11-82e5-000d3ab87733",
+            "ownerid": "7d48edd3-69fd-ec11-82e5-000d3ab87733"
+          }
+        }
+      ]
+    }
+  };
+  //#endregion
+
+  let log: string[];
+  let logger: Logger;
+  let loggerLogSpy: sinon.SinonSpy;
+
+  before(() => {
+    sinon.stub(auth, 'restoreAuth').callsFake(() => Promise.resolve());
+    sinon.stub(appInsights, 'trackEvent').callsFake(() => { });
+    sinon.stub(pid, 'getProcessName').callsFake(() => '');
+    auth.service.connected = true;
+  });
+
+  beforeEach(() => {
+    log = [];
+    logger = {
+      log: (msg: string) => {
+        log.push(msg);
+      },
+      logRaw: (msg: string) => {
+        log.push(msg);
+      },
+      logToStderr: (msg: string) => {
+        log.push(msg);
+      }
+    };
+    loggerLogSpy = sinon.spy(logger, 'log');
+  });
+
+  afterEach(() => {
+    sinonUtil.restore([
+      request.get
+    ]);
+  });
+
+  after(() => {
+    sinonUtil.restore([
+      auth.restoreAuth,
+      appInsights.trackEvent
```

Comment:
Hi @appieschot seems like your pipeline is failing cause of the added process stub. You should also add `pid.getProcessName` here.